frontstage/views/passwords.py

```diff
@@ -24,6 +25,46 @@ def forgot_password():
         email_address = request.form.get('email_address')
         post_data = {"email_address": email_address}
 
+        try:
```

Comment:
There's probably a bunch of reused code between this and the the sign-in page. Could look at refactoring it together in the move to frontstage-api

Reply:
Can do the refactoring as part of this ticket if it makes the switch to the frontage-api easier?

Reply:
I'm porting the sign-in stuff over already right now so probably easier if I do it :+1: